db/models.py

```diff
@@ -14,3 +14,44 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True)
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="movies")
```

Comment:
I think it's bad related_name, better use `actors` and `genres` names, for example

Reply:
very my bad, sorry. your solution is correct